src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -209,6 +209,63 @@ def test_storage_create_with_encryption_key_type(self, resource_group):
             JMESPathCheck('encryption.services.table.keyType', 'Account'),
         ])
 
+    @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2019-06-01')
+    @ResourceGroupPreparer(location='eastus', name_prefix='cli_storage_account_routing')
+    def test_storage_account_with_routing_preference(self, resource_group):
```

Comment:
Could you add some cases where `--publish-internet-endpoints` or `--publish-microsoft-endpoints` is false?

Reply:
Added